config/application.rb

```diff
@@ -140,6 +140,7 @@ class Application < Rails::Application
 
       # To evaluate settings or database.yml with encrypted passwords
       require 'manageiq-password'
+      require 'manageiq/password/password_mixin'
```

Comment:
Trying to remove the need for this via https://github.com/ManageIQ/manageiq-password/pull/13

Reply:
cool, this was merged so you can drop this line now.  👍

Reply:
Not until I release the gem.  I'm thinking let's merge as is, and then when I cut a major version of the gem it will require a bunch of other change anyway, and I'll include this one with it.